pom.xml

```diff
@@ -109,6 +109,8 @@
         <rxjava.version>2.1.9</rxjava.version>
         <commons-codec.verion>1.10</commons-codec.verion>
         <jmh.version>1.21</jmh.version>
+        <httpcore5.version>5.0-beta10</httpcore5.version>
+        <httpclient5.version>5.0-beta7-SNAPSHOT</httpclient5.version>
```

Comment:
One should not rely on snapshots. There is no guarantee that someone can really pull them.

Reply:
There's a show-stopping bug in beta6, so I had to use a local snapshot temporarily.

Reply:
@rschmitt @michael-o We will get another BETA (the last one) or GA soon.